rules/rules_scala.bzl

```diff
@@ -56,12 +56,7 @@ emulate_rules_scala_repository = repository_rule(
     implementation = _emulate_rules_scala_repository_impl,
 )
 
-def emulate_rules_scala(scala, scalatest, extra_deps = []):
-    native.bind(
```

Comment:
Do we need to replace this with anything? I imagine this is because the version of rules_scala from the compat layer is so old it's still using bind? I imagine it's not worth upgrading at this point, but I figured I'd ask.

Reply:
Nope! That was used here, but it's no longer used:
https://github.com/lucidsoftware/rules_scala/pull/51/files#diff-471416c47abcd84fb475b6b41ec3ed263aff9093ffd10f9f807fb9e38ddb09acL26